kyuubi-server/web-ui/src/views/management/engine/index.vue

```diff
@@ -151,6 +163,10 @@
         getList()
       })
   }
+  function openEngineUi(url: string) {
+    url = (url || '').replaceAll(/http:|https:/gi, '')
+    window.open(`${import.meta.env.VITE_APP_DEV_WEB_URL}engine-ui/${url}/`)
```

Comment:
how does this work?

suppose `url=http://test.org:1234` and `VITE_APP_DEV_WEB_URL=http://kyuubi/ui/` what's the expected result?

Reply:
it will call kyuubi server with `http://test.org:1234/engine-ui/host:ip/job`, kyuubi server proxy handle will pass request to engine ui.

Reply:
based on my understanding, it actual result is `http://test.org:1234/engine-ui///host:ip`

Reply:
let's supply a UT for it